CMakeLists.txt

```diff
@@ -193,6 +193,11 @@ if(APPLE)
     # CoreML MLCustomModel requires macOS 10.14 or higher
     set(HAS_MLCUSTOMMODEL TRUE)
   endif()
+
+   # MPSForward Loss is only present on macOS 10.15 or higher.
```

Comment:
I don't think we need this comment, as the new preprocessor symbol is not specific to `MPSForwardLoss`

Reply:
Removed comment!